sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/samples/Generated/Samples/Sample_ConnectedClusterServiceCollection.cs

```diff
@@ -21,7 +21,7 @@ public partial class Sample_ConnectedClusterServiceCollection
         public async Task CreateOrUpdate_ServicesCreateOrUpdate()
         {
             // Generated from example definition: 2024-03-01/Services_CreateOrUpdate.json
-            // this example is just showing the usage of "ServiceResource_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Services_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/kubernetesruntime/KubernetesRuntime.Management/service.tsp#L35